src/api/app/controllers/webui/project_controller.rb

```diff
@@ -1004,16 +1001,12 @@ def check_devel_package_status(currentpack, p)
     currentpack['develproject'] = dproject
     currentpack['develpackage'] = dp.name
     key = "#{dproject}/#{dp.name}"
-    if @submits.has_key? key
-      currentpack['requests_to'].concat(@submits[key])
-    end
+    currentpack['requests_to'].concat(@submits[key]) if @submits.has_key? key
```

Comment:
[nitpick]
Guess it is autocorrected but what about parentheses?
```@submits.has_key?(key)```

(https://github.com/bbatsov/rubocop/issues/3224 ``has_key?`` seems to be "deprecated")